es-app/src/guis/GuiMenu.cpp

```diff
@@ -23,381 +23,412 @@
 
 GuiMenu::GuiMenu(Window* window) : GuiComponent(window), mMenu(window, "MAIN MENU"), mVersion(window)
 {
-	// MAIN MENU
+	if (ViewController::get()->isUIModeFull())
+	{
+		//auto openScrapeNow = [this] { mWindow->pushGui(new GuiScraperStart(mWindow)); };
+		addEntry("SCRAPER", 0x777777FF, true, [this] { openScraperSettings(); });
+
+		addEntry("SOUND SETTINGS", 0x777777FF, true, [this] {openSoundSettings(); });
+
+		addEntry("UI SETTINGS", 0x777777FF, true, [this] {openUISettings(); });
+
+		addEntry("GAME COLLECTION SETTINGS", 0x777777FF, true, [this] { openCollectionSystemSettings(); });
+
+		addEntry("OTHER SETTINGS", 0x777777FF, true, [this] {openOtherSettings(); });
+
+		addEntry("CONFIGURE INPUT", 0x777777FF, true, [this] {openConfigInput(); });
+	}
+
+	addEntry("QUIT", 0x777777FF, true, [this] {openQuitMenu(); });
```

Comment:
Whitespace inconsistency; space before curly bracket missing in some of these >  `[this] {openXXX(); }`

Reply:
noted